spec/presenters/candidate_interface/application_form_presenter_spec.rb

```diff
@@ -75,6 +75,40 @@
     end
   end
 
+  describe '#other_qualifications_completed?' do
+    it 'returns true if other qualifications section is completed' do
+      application_form = FactoryBot.build(:application_form, other_qualifications_completed: true)
+      presenter = CandidateInterface::ApplicationFormPresenter.new(application_form)
+
+      expect(presenter).to be_other_qualifications_completed
```

Comment:
the `to be_other_qualifications_completed` is not so readable, what if we keep it simple? :

`expect(presenter.other_qualifications_completed?).to be true`

Reply:
Updated!